builders/build/eth-api/dev-env/foundry.md

```diff
@@ -99,13 +99,17 @@ forge install OpenZeppelin/openzeppelin-contracts
 forge build
 ```
 
-![Foundry Contract Compile](/images/builders/build/eth-api/dev-env/foundry/foundry-1.webp)
+--8<-- 'code/builders/build/eth-api/dev-env/foundry/terminal/compile.md'
```

Comment:
Closest I could get to comment but line 88 `Github repo` should be changed to `GitHub repository` (note the capital H in GitHub)